device-registry/src/main/scala/org/genivi/sota/device_registry/GroupsResource.scala

```diff
@@ -35,15 +36,22 @@ class GroupsResource(namespaceExtractor: Directive1[Namespace])
             _       <- GroupMember.createGroup(groupId, namespace, request.device1)
             _       <- GroupMember.createGroup(groupId, namespace, request.device2)
                  //PRO-1378 Add logic to find devices which should be in this group
-          } yield ()
+          } yield groupId
```

Comment:
No, this needs to be returned as group queries require the use of the groupId, and if you just give a name when you make the request, then you won't know what the id is. It's used in the tests for example.